src/tutorials/0006/06.md

```diff
@@ -0,0 +1,46 @@
+You can paste any IPFS CID into the handy [CID Inspector](http://cid.ipfs.io/) to visualize all of its prefixes and what they represent.
+
+In this final lesson we will take a look at some results from this tool using both CIDv0 and CIDv1 formats.
+
+## Example 1: CIDv1
+
+[`bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi`](https://cid.ipfs.io/#bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi)
+
+This first example is a version 1 CID.
+
+![Results from the CID Inspector tool for the first example](tutorial-assets/T0006L06-example-1.jpg)
+
+Looking at the results from the CID Inspector tool we can see several parts that the tool was able to parse for us:
+
+- `Human Readable CID`: breaks down each part of the CID to be easily readable by us humans
+- `Multibase`: `code` is the identifier of the base. `b` is the id for `base32`.
+- `Multicodec`: `code` is the identifier of the codec. `0x70` is the id for `dag-pb`.
+- `Multihash`: breakdown of the multihash, `18` is the id for the `sha2-256` hashing algorithm, and `bits=256` is the length of the hash
+
+From the "Human Readable CID" breakdown, we can see that the original hash of the content, before the appropriate CIDv1 prefixes are added, is `c3c4733ec8affd06cf9e9ff50ffc6bcd2ec85a6170004bb709669c31de94391a`.
+
+## Example 2: CIDv0
+
+[`QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR`](https://cid.ipfs.io/#QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR)
+
+![Results from the CID Inspector tool for the first example](tutorial-assets/T0006L06-example-2.jpg)
+
+This Version 0 CID shows some different results: both the `multibase` and the `multicodec` are listed as "implicit".
+Since Version 0 CIDs did not have those prefixes, they are always assumed to be `base58btc` and `dag-pb` respectively.
+
+You may have noticed that the `Base32 CIDV1` part in the end is `bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi`, which is the same CID from the first example!
+
+Notice also how the end of the "Human Readable CID" (the portion after the prefixes) is exactly the same in this CIDv0 example as it was in the CIDv1 example: `c3c4733ec8affd06cf9e9ff50ffc6bcd2ec85a6170004bb709669c31de94391a`.
+
+Why? These two CIDs point to the same content. Basically, it's the same hash (`c3c4733ec8affd06cf9e9ff50ffc6bcd2ec85a6170004bb709669c31de94391a`) represented in the two different versions of the CID spec.
+
+You can convert any `CIDv0` to `CIDv1`, because the implicit prefixes from `v0` become explicit in `v1`.
+However, because `CIDv1` supports multiple codecs and multiple bases and `CIDv0` does not, not all `CIDv1` hashes can be converted to `CIDv0`. In fact, only `CIDv1` that have `multibase=base58btc` and `multicodec=dag-pb` can be converted to `CIDv0`.
```

Comment:
...and hash algo sha2-256 and hash length 256

https://github.com/multiformats/js-cid/blob/9fee0105e300c8e7cb9d14cda6823b62d61982eb/src/index.js#L192-L208